.github/workflows/wheel.yml

```diff
@@ -0,0 +1,423 @@
+          uses: docker/setup-qemu-action@49b3bc8e6bdd4a60e6116a5414239cba5943d3cf # v3.2.0
+          with:
+            platforms: all
+
+        - name: Build wheels
+          uses: pypa/cibuildwheel@d4a2945fcc8d13f20a1b99d461b8e844d5fc6e23 # v2.21.1
+          env:
+            CIBW_BUILD: ${{ matrix.python }}
+            CIBW_ARCHS: ${{ matrix.arch }}
+            CIBW_MANYLINUX_AARCH64_IMAGE: ${{ matrix.manylinux }}
+            # Apparently, the x86_64 runners aren't able to execute tests
+            # tests for ARM wheels. 
+            # TODO: Re-enable tests when linux ARM runners are available
+            CIBW_TEST_SKIP: '*'
+
+        - uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4.4.0
+          with:
+            name: cibw-wheels-${{ matrix.python }}-${{ matrix.manylinux }}
+            path: ./wheelhouse/*.whl
+
+  # ---------------------------------------------------------------------------
+  # macOS Wheels
+  # ---------------------------------------------------------------------------
+
+  macos:
+    name: Build wheels on macOS
+    runs-on: macos-12
+    # Don't run on OIIO forks
+    if: |
+      github.event_name != 'schedule' ||
+      github.repository == 'AcademySoftwareFoundation/OpenImageIO'
+    strategy:
+      matrix:
+        include:
+          # -------------------------------------------------------------------
+          # CPython 64 bits
+          # -------------------------------------------------------------------
+          - build: CPython 3.8 64 bits
+            python: cp38-macosx_x86_64
+            arch: x86_64
+          - build: CPython 3.9 64 bits
+            python: cp39-macosx_x86_64
+            arch: x86_64
+          - build: CPython 3.10 64 bits
+            python: cp310-macosx_x86_64
+            arch: x86_64
+          - build: CPython 3.11 64 bits
+            python: cp311-macosx_x86_64
+            arch: x86_64
+          - build: CPython 3.12 64 bits
+            python: cp312-macosx_x86_64
+            arch: x86_64
+          - build: CPython 3.13 64 bits
+            python: cp313-macosx_x86_64
+            arch: x86_64
+
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      - uses: actions/setup-python@f677139bbe7f9c59b41e40162b753c062f5d49a3 # v5.2.0
+        name: Install Python
+        with:
+          python-version: '3.9'
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@d4a2945fcc8d13f20a1b99d461b8e844d5fc6e23 # v2.21.1
+        env:
+          CIBW_BUILD: ${{ matrix.python }}
+          CIBW_ARCHS: ${{ matrix.arch }}
+          # TODO: Re-enable HEIF when we provide a build recipe that does 
+          # not include GPL-licensed dynamic libraries.
+          USE_Libheif: 'OFF'
+          # Make sure the OIIO build's version of freetype is linked
+          CIBW_BEFORE_ALL: |
+            brew uninstall --ignore-dependencies harfbuzz fontconfig cairo freetype
```

Comment:
It would be better to force CMake to ignore brew's provided dependencies. I thnink this can be achieved using https://cmake.org/cmake/help/latest/variable/CMAKE_FIND_USE_CMAKE_SYSTEM_PATH.html#variable:CMAKE_FIND_USE_CMAKE_SYSTEM_PATH (see https://cmake.org/cmake/help/latest/command/find_package.html#index-0-variable:CMAKE_INSTALL_PREFIX).

Reply:
Interesting... I'll look into both of these approaches. Thanks for the tips!
Seems like it should just be a matter of appending /opt/homebrew to CMAKE_IGNORE_PATH...

Reply:
I added a IGNORE_HOMEBREWED_DEPS option which appends homebrew prefixes to the CMAKE_IGNORE_PATH. I also made a slight modification to the src/cmake/dependency_utils.cmake:build_dependency_with_cmake, to make sure dependency builds inherit CMAKE_IGNORE_PATH too. It seems to work alright.